tests/test_scheduler.py

```diff
@@ -2190,3 +2176,61 @@ def test_scheduler_unqualified_imports(config):
 
     assert item.enable_imports
     assert item.children == ('other_routine',)
+
+
+def test_transformation_config(config):
+    """
+    Test the correct instantiation of :any:`Transformation` objecst from config
+    """
+    my_config = config.copy()
+    my_config['transformations'] = {
```

Comment:
Can we parametrise this test to provide `my_config['transformations']` in the two supported formats? I'm not quite clear what the second is meant to be here:
https://github.com/ecmwf-ifs/loki/blob/5899bf00a6f969c762e6c7551b7141b30dbaaf02/loki/batch/configure.py#L57-L60

Reply:
Nah, I think I just forgot to remove this. Fixed now.